<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(31, 31, 31)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="0">
     <widget class="QLabel" name="labelMenuGraCredits">
      <property name="styleSheet">
       <string notr="true">color: white</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButtonlogout">
      <property name="styleSheet">
       <string notr="true">color: white</string>
      </property>
      <property name="text">
       <string>logout</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="labelMenuGraUserName">
      <property name="styleSheet">
       <string notr="true">color: white</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="MenuGra">
       <widget class="QPushButton" name="pushButtonGra2">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>100</y>
          <width>191</width>
          <height>151</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: white;
background-color: rgb(45, 53, 129)</string>
        </property>
        <property name="text">
         <string>Gra2</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonGra1">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>100</y>
          <width>191</width>
          <height>151</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: white;
background-color: rgb(87, 145, 71)</string>
        </property>
        <property name="text">
         <string>Gra1</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="Gra1SlotMachine">
       <widget class="QPushButton" name="spinButton">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>270</y>
          <width>101</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: white;
background-color: rgb(125, 57, 57);</string>
        </property>
        <property name="text">
         <string>SPIN</string>
        </property>
       </widget>
       <widget class="QLabel" name="slotMachineDrum1">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>50</y>
          <width>175</width>
          <height>197</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QLabel" name="slotMachineDrum2">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>50</y>
          <width>175</width>
          <height>197</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QLabel" name="slotMachineDrum3">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>50</y>
          <width>175</width>
          <height>197</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="betSlotMachine">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>270</y>
          <width>101</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>11</pointsize>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 11pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
background-color: rgb(74, 111, 4)</string>
        </property>
       </widget>
       <widget class="QPushButton" name="BackToMainMenu">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>270</y>
          <width>101</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: white; background-color: rgb(137, 144, 63)</string>
        </property>
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="Gra2">
       <widget class="QPushButton" name="BackToMainMenu_2">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>10</y>
          <width>91</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 11pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
background-color: rgb(137, 144, 63)</string>
        </property>
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_Millionaire_Pytanie">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>70</y>
          <width>481</width>
          <height>41</height>
         </rect>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 11pt &quot;Segoe UI&quot;;
font: 700 9pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
background-color: rgb(12, 29, 121);
text-align: center;</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_Millionaire_ODP_A">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>130</y>
          <width>481</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
background-color: rgb(139, 73, 157)</string>
        </property>
        <property name="text">
         <string>Odpowiedz A</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_Millionaire_ODP_B">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>180</y>
          <width>481</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
background-color: rgb(139, 73, 157)</string>
        </property>
        <property name="text">
         <string>Odpowiedz B</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_Millionaire_ODP_C">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>230</y>
          <width>481</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
background-color: rgb(139, 73, 157)</string>
        </property>
        <property name="text">
         <string>Odpowiedz C</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_Millionaire_ODP_D">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>280</y>
          <width>481</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
background-color: rgb(139, 73, 157)</string>
        </property>
        <property name="text">
         <string>Odpowiedz D</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="textEdit_MillionaireBET">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>10</y>
          <width>71</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 11pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
background-color: rgb(74, 111, 4)</string>
        </property>
       </widget>
       <widget class="QPushButton" name="on_pushButton_Milionaire_Start_clicked">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>10</y>
          <width>81</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: white;
background-color: rgb(125, 57, 57);</string>
        </property>
        <property name="text">
         <string>START</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="Start">
       <widget class="QTextEdit" name="textLogin">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>10</y>
          <width>241</width>
          <height>61</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: white</string>
        </property>
        <property name="midLineWidth">
         <number>4</number>
        </property>
        <property name="markdown">
         <string>Login

</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonRegister">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>220</y>
          <width>241</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 11pt &quot;Segoe UI&quot;;
color: white;
background-color: rgb(44, 30, 148)</string>
        </property>
        <property name="text">
         <string>Register</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonLogIn">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>160</y>
          <width>241</width>
          <height>51</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="palette">
         <palette>
          <active>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>8</red>
              <green>115</green>
              <blue>29</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="ButtonText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>8</red>
              <green>115</green>
              <blue>29</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>8</red>
              <green>115</green>
              <blue>29</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="PlaceholderText">
            <brush brushstyle="SolidPattern">
             <color alpha="128">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>8</red>
              <green>115</green>
              <blue>29</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="ButtonText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>8</red>
              <green>115</green>
              <blue>29</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>8</red>
              <green>115</green>
              <blue>29</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="PlaceholderText">
            <brush brushstyle="SolidPattern">
             <color alpha="128">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>8</red>
              <green>115</green>
              <blue>29</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="ButtonText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>8</red>
              <green>115</green>
              <blue>29</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>8</red>
              <green>115</green>
              <blue>29</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="PlaceholderText">
            <brush brushstyle="SolidPattern">
             <color alpha="128">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 11pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
background-color: rgb(8, 115, 29)</string>
        </property>
        <property name="text">
         <string>Log In!</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="textPassword">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>80</y>
          <width>241</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:white;
font: 11pt &quot;Segoe UI&quot;;</string>
        </property>
        <property name="inputMask">
         <string/>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="pushButton_exit">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white</string>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
